test/sql/pragma/test_pragma_version.test

```diff
@@ -0,0 +1,15 @@
+# name: test/sql/pragma/version.test
+# description: Test version pragma
+# group: [pragma]
+
+statement ok
+PRAGMA version;
+
+statement ok
+select * from pragma_version();
+
+query I
+select library_version from pragma_version();
```

Comment:
It would be nice to test, the source_id column as well, but I think that would change on every commit making this test fail, a imperfect approximation could be to check that the source_id column is not null, or matches a regex of alphanumeric characters of a certain length.